src/Components/RecordIssues/index.tsx

```diff
@@ -81,31 +81,28 @@ export default class RecordIssues extends React.Component<IRecordIssueProps, IRe
 
     renderAddIssuesButton(){
         return (
-            <div style={ this.props.issues.length > 0 ? divStyle : {}}>
-                <button 
-                    id="add-issue"
-                    data-test="add-issue"
-                    className="button btn-primary btn-stacked"
-                    onClick={this.addIssueComponent}>
-                        Add an issue
-                </button>
-            </div>
-        );
-    }
-
-    renderSavingSignMeetingText(){
-        return (
-        <div>
-            <div className="heading">Saving and signing off the ETRA meeting</div>
-            <p>The ETRA meeting needs to synchronise with Manage a tenancy. You need to be online and signed into Manage a tenancy 
-                for this to happen. Do not close your web browser or switch off your iPad until you have done this.</p>
-            <p>If you are offline or not signed into Manage a tenancy you can only ‘Save the issues for review with the TRA later’. The TRA 
-                representative’s signature, name and role will not be saved at this stage. Once you are back online and signed in, you will be 
-                able to ‘Email issues to the TRA for sign off’. To do this you need sign in to Manage a tenancy and navigate to the ETRA meetings 
-                page and click on the link: ‘Draft ETRA meetings for review’. You should then see a link to the meeting on the next page..</p>
-            <p>If at the meeting you are online, sign into Manage a tenancy and then the TRA representative can sign, add 
-            their name and role and you can ‘Save the signed off issue list and email to TRA’.</p>
-        </div>
+            <>
```

Comment:
Should this be opening <div> tag perhaps?

Reply:
Its a React "Fragment" tag. It just lets me stick together 2 separate div elements for rendering. I think I would have to replace the fragment with a div tag to get the same behaviour from React.